backend/src/database/seeds/05-Top_Rope_Grades.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex';
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries and reset primary key
+  await knex.raw('TRUNCATE TABLE top_rope_grades RESTART IDENTITY CASCADE');
+
+  // Inserts seed entries
+  await knex('topRopeGrades').insert([
+    { name: '4' },
+    { name: '5c' },
+    { name: '5c+' },
```

Comment:
There's no 5c+. Can remove.

Fyi, for full list of grades see here. https://www.guidedolomiti.com/en/rock-climbing-grades/

Though we don't have to follow it exactly. Current one is good enuf.